modules/release-notes/pages/relnotes.adoc

```diff
@@ -122,13 +195,59 @@ SELECT ... FROM ... WHERE ...
 For example, the modulo operation or a scan for min int64 value returns an incorrect result in case of int64.
 |===
 
-*Security*
+==== Security
 
-[#table_knownissues_v55-security,cols="25,66"]
+[#table_knownissues_v600-security,cols="25,66"]
 |===
 | Issue | Description
 
+| https://issues.couchbase.com/browse/MB-31558[MB-31558^]
+| *Summary*: All requests originating from the UI are currently authenticated using token based authentication, even when client certificate authentication is enabled on the Server and a client certificate is configured in the browser.
+
 | https://issues.couchbase.com/browse/MB-26421[MB-26421^]
-| *Summary*: The default "administrator" user doesn't show up in list of local users displayed on the Web Console under the Security \-> Users tab.
+| *Summary*: The Security > Users tab in the UI does not list the default "administrator" user. 
+|===
+
+==== Tools, Web Console (UI), and REST API
+
+[#table_knownissues_v600-tools-ui-rest-api,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-31680[MB-31680^]
+| *Summary*: Unable to log in to the Couchbase Server Web Console (UI) using Internet Explorer 11 or Edge browsers.
+|===
+
+[#fixed-issues-600]
+=== Fixed Issues
+
+The notable fixes in this release are listed below. See the Couchbase https://issues.couchbase.com/issues/?filter=16883[issue tracker^] for the complete list of enhancements and fixes in this release.
+
```

Comment:
Please add an entries for the following MBs:

https://issues.couchbase.com/browse/MB-31495:
*Summary*: Fetching a random key from the Data Service may hang if the bucket contains zero documents.

https://issues.couchbase.com/browse/MB-31175:
*Summary*: Ephemeral buckets can have tombstones purged before the configured metadata purge interval.
Ephemeral buckets can prematurely purge (discard) tombstones when under high memory and op/s load. As a consequence DCP consumers fed by KV-Engine (such as replication, secondary indexing, views) may have tombstones not sent to them, meaning their dataset diverges from the Data Service - documents are not deleted from the DCP consumer.

https://issues.couchbase.com/browse/MB-31481:
*Summary*: KV-Engine may not send STREAM_END message to consumers if non-infinity end seqno and cursor dropping occurs.

https://issues.couchbase.com/browse/MB-31619:
*Summary*: memcached logs always use Z (UTC) for timezone when logging instead of current system timezone offset.

Reply:
Added